src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import Button, { ButtonProps } from "./Button"
+import { action } from "@storybook/addon-actions"
+import { Meta, Story } from "@storybook/react/types-6-0"
+
+export default {
+  title: "Design System/Button",
+  component: Button,
+} as Meta
+
+const Template: Story<ButtonProps> = (props: ButtonProps) => {
+  return <Button {...props} />
+}
+
+export const Example = Template.bind({})
+Example.args = {
+  children: "Submit",
+  disabled: false,
+  variant: "filled",
+  color: "primary",
+  onClick: action("clicked"),
```

Comment:
Usually you don't have to add this, as the default storybook config detects any prop that matches the regex "^on[A-Z].*" as an action, and tracks it automatically